nat-lab/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "nat-lab"
+
+[tool.black]
+preview = true
+skip_magic_trailing_comma = true
+exclude="tests/protobuf"
+
+[tool.isort]
+profile = "black"
+force_single_line = false
+no_inline_sort = true
+sort_reexports = true
+float_to_top = true
+combine_star = true
+remove_redundant_aliases = true
+combine_as_imports = true
+no_sections = true
+skip=["tests/protobuf"]
+
+[tool.autoflake]
```

Comment:
What is `tool.autoflake`? I don't see it used in CI. Is it implicitly used by pylint or mypy?

Reply:
I just added for myself, will add it for CI too... don't know why I didn't :smile:

Reply:
added to CI also